packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -159,6 +164,8 @@ export default function EntitiesSavedStates( { close } ) {
 				siteItemsToSave
 			);
 		}
+
+		__unstableMarkLastChangeAsPersistent();
```

Comment:
Maybe instead of dispatching this here, we could do so inside the `close` callback so we don't dispatch this action on every instance of `EntitiesSavedState`. 

But on the other hand, it makes sense to dispatch this action here since it's part of editing save.